src/modules/Experience/scenes/MediaCoverageScene/MediaCoverageScene.tsx

```diff
@@ -1,40 +1,89 @@
-import React, { useRef, useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { useFrame, Vector3 } from '@react-three/fiber';
 import { Text } from '@react-three/drei';
+import { MediaCard } from './components';
+import { useAppBreakpoints } from '@/hooks';
+// import { MediaCard } from './components';
```

Comment:
Lets increase the level of PR! 
Unused imports should go. :D . You should start doing the same for me :-P